### PR TITLE
Open election tests

### PR DESCRIPTION
**Pull request by Tesa27**
- This PR addresses first of all tests for open election message and adapts the flow from election setup to election end in the current karate tests.
- It also constructs an API that build different ids ( here vote ids, election ids and question ids) since we do not want to have them constant and not know how they where computed, this way if something changes in the computation it will be much easier to adapt, and the errors if they happen are easier to spot.

### CODE REVIEW

File: tests/karate/src/test/java/be/constants.feature [hunk 1]
```diff
@@ -126,10 +158,30 @@ Feature: Constants
...
+      """
+        function(){
+          return createIsThisProjectFunVoteId("0")
+        }
+      """
+    * def createIsThisProjectFunVoteIdVoteNo =
       """
         function(){
-          return "d60B94lVWm84lBHc9RE5H67oH-Ad3O1WFflK3NSY3Yk="
+          return createIsThisProjectFunVoteId("1")
```
Comment (GabrielFleischer) [inline tests/karate/src/test/java/be/constants.feature:184]: I am not particularly up to date with the spec, but what is the 1 supposed to mean ?

File: tests/karate/src/test/java/be/election/electionOpen.feature [hunk 2]
```diff
@@ -0,0 +1,122 @@
+
+CODE CLIMATE OPTIONS
+Show issues
+Show test coverage
+Highlight covered lines
```
Comment (GabrielFleischer) [inline tests/karate/src/test/java/be/election/electionOpen.feature:5]: What is this ?

**Review by GabrielFleischer — DISMISSED** [review]
LGTM
I just have a small question o the election. But this is just my lack of knowledge concerning the election protocol.

File: tests/karate/src/test/java/be/constants.feature (hunk 1 above)
Comment (Tesa27) [reply tests/karate/src/test/java/be/constants.feature:184]: 1 is the index of the corresponding ballot option

Comment (Tesa27) [reply tests/karate/src/test/java/be/constants.feature:184]: Here 0 or 1 would correspond for voting yes or no respectively which are the ballot options available

File: tests/karate/src/test/java/be/election/electionOpen.feature (hunk 2 above)
Comment (Tesa27) [reply tests/karate/src/test/java/be/election/electionOpen.feature]: I do not have a slightest idea, I removed it now

**Review by GabrielFleischer — APPROVED** [review]
LGTM

**Pull request merged**